src/__.js

```diff
@@ -1 +1,26 @@
+/**
+ * A special placeholder value used to specify "gaps" within curried functions,
+ * allowing partial application of any combination of arguments,
+ * regardless of their positions.
+ *
+ * If `g` is a curried ternary function and `_` is `R.__`, the following are equivalent:
+ *
+ *   - `g(1, 2, 3)`
+ *   - `g(_, 2, 3)(1)`
+ *   - `g(_, _, 3)(1)(2)`
+ *   - `g(_, _, 3)(1, 2)`
+ *   - `g(_, 2, _)(1, 3)`
+ *   - `g(_, 2)(1)(3)`
+ *   - `g(_, 2)(1, 3)`
+ *   - `g(_, 2)(_, 3)(1)`
+ *
+ * @constant
+ * @memberOf R
+ * @category Function
+ * @example
+ *
+ *      var powThenMod = R.curry(function(base, exp, mod) { return Math.pow(base, exp) % mod; });
+ *      var cubeThenMod = powThenMod(R.__, 3, R.__);
+ *      cubeThenMod(4, 10); //=> 4
```

Comment:
The example above is a bit contrived. I'd prefer something like this:

```javascript
var greet = R.replace('{name}', R.__, 'Hello, {name}!');

greet('Alice'); //=> 'Hello, Alice!'
```

Ramda's build script uses this approach [here][1].


[1]: https://github.com/ramda/ramda/blob/a6b5af388741b6a36a786bd39dde78e8dc321fdc/scripts/build#L176